meta-openpli/recipes-openpli/enigma2/enigma2.bb

```diff
@@ -113,6 +113,7 @@ PKGV = "2.7+git${GITPKGV}"
 ENIGMA2_BRANCH ?= "develop"
 GITHUB_URI ?= "git://github.com"
 SRC_URI = "${GITHUB_URI}/OpenPLi/${BPN}.git;branch=${ENIGMA2_BRANCH}"
+SRC_URI_append_libc-musl = " file://0001-enigma2-fix-build-with-musl.patch"
```

Comment:
@andrea-adami Would you please send a PR for https://github.com/OpenPLi/enigma2 or @WanWizard  could apply this patch.